packages/documentation/src/shared/nb-bootstrap/ngb-component-demo-link.mdx

```diff
@@ -1,8 +1,15 @@
-<a
-  className="btn btn-rg btn-secondary btn-animated"
-  href={`https://archive.design-system.post.ch/#/ng-bootstrap-samples/${props.component}`}
-  target="_blank"
-  rel="nofollow noopener noreferrer"
->
-  <span>See archived documentation</span>
-</a>
+[//]: # (<a)
```

Comment:
I commented this link to avoid duplicates: 
![image](https://github.com/swisspost/design-system/assets/33580481/1dfdb265-d252-47b9-816f-601c5d8a60b9)